ui/src/app/routing/guards.tsx

```diff
@@ -46,8 +47,7 @@ export const registrationGuard: Guard = {
 };
 
 export const expiredGuard: Guard = {
-  allowed: (): boolean =>
-    !profileStore.get().profile.accessModules.anyModuleHasExpired,
```

Comment:
`anyModuleHasExpired` is an old concept from the early days of AAR, and has not kept up with our logic.  We should eventually remove it from the API.